package.json

```diff
@@ -2,18 +2,19 @@
   "name": "ethereumjs-wallet",
   "version": "0.6.1",
   "description": "Utilities for handling Ethereum keys",
-  "main": "dist/index.js",
```

Comment:
Alternatively, you could keep this as `dist/index.js`, build from `src/` into `dist/`, and place a script/s that doesn't get transpiled by babel at the top-level of the package, e.g. `hdkey.js`:

``` js
module.exports = require('./dist/hdkey.js');
```